lib/document-locale-settings.js

```diff
@@ -0,0 +1,140 @@
+const host = document.documentElement;
+
+const DEFAULTS = {
+	pseudoLocalization: { isAvailable: false, isEnabledForUser: false, textFormat: '{0}' },
```

Comment:
Do we actually need to send down `isAvailable` and `isEnabledForUser`, or could we just not even set the `data-` attribute if both those things aren't true? Like those are server-side things that it doesn't feel like the front end needs to know about...

Reply:
Yeah how we do that is totally up in the air. What you suggest makes sense to me. I mostly just put everything in here that I think we ultimately _might_ need (even if only on the server). So, I didn't include any of the character look-alike stuff etc.